.github/workflows/ci.yml

```
@@ -60,10 +60,10 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Build
-        run: make
+        run: make -f ${{ runner.os == 'Windows' && 'Makefile.win' || 'Makefile' }}
```

I'd suggest using ternary operator as it's more straightforward.

```suggestion
        run: make -f ${{ runner.os == 'Windows' ? 'Makefile.win' : 'Makefile' }}
```

---

[They are not available](https://github.com/orgs/community/discussions/75928)

---

Wow, GHA weirdness make my head spin sometimes 🤯 Thanks for the clarification.